thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -577,7 +578,8 @@ public Response tuneAlertFilter(@PathParam("functionId") long id,
       @QueryParam("end") String endTimeIso,
       @QueryParam("autoTuneType") @DefaultValue("AUTOTUNE") String autoTuneType,
       @QueryParam("holidayStarts") @DefaultValue("") String holidayStarts,
-      @QueryParam("holidayEnds") @DefaultValue("") String holidayEnds) {
+      @QueryParam("holidayEnds") @DefaultValue("") String holidayEnds,
+      @QueryParam("tuningFeatures") String features) {
```

Comment:
Please add explanation in javadoc for features. Thanks!

Reply:
Added. Thanks for the comment.

Reply:
Added! Thanks!